presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -204,7 +204,7 @@ public void testCreateTable()
         assertTableColumnNames(tableName, "a", "b", "c");
 
         String tableNameLike = "test_create_like_" + randomTableSuffix();
-        assertUpdate("CREATE TABLE " + tableNameLike + " (LIKE " + tableName + ", d boolean, e varchar)");
```

Comment:
please add boolean to `io.prestosql.testing.AbstractTestDistributedQueries#testDataMappingSmokeTestData` (can be a follow up)

Reply:
https://github.com/prestosql/presto/pull/5900